src/main/drivers/inverter.c

```diff
@@ -20,51 +20,64 @@
 
 #include "platform.h"
 
+#include "io/serial.h" // For SERIAL_PORT_IDENTIFIER_TO_RESOURCE_INDEX
 #include "drivers/io.h"
-#include "io_impl.h"
+#include "drivers/serial.h"
+
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
 
 #include "inverter.h"
 
 #ifdef USE_INVERTER
+
+static const serialPinConfig_t *pSerialPinConfig;
+
 static void inverterSet(IO_t pin, bool on)
 {
     IOWrite(pin, on);
 }
 
-static void initInverter(ioTag_t ioTag)
+static void initInverter(int index)
 {
-    IO_t pin = IOGetByTag(ioTag);
-    IOInit(pin, OWNER_INVERTER, 1);
+    IO_t pin = IOGetByTag(pSerialPinConfig->ioTagInverter[index]);
+    IOInit(pin, OWNER_INVERTER, RESOURCE_INDEX(index));
     IOConfigGPIO(pin, IOCFG_OUT_PP);
 
     inverterSet(pin, false);
 }
 #endif
 
-void initInverters(void)
+void initInverters(const serialPinConfig_t *serialPinConfigToUse)
 {
+#ifdef USE_INVERTER
+    pSerialPinConfig = serialPinConfigToUse;
+
 #ifdef INVERTER_PIN_UART1
```

Comment:
`#ifdef` is not necessary - user may set  inverter pin even when it is not set by default 

but `USE_UARTx` should be probably checked. It may be worth defining `USED_UARTS` similar to `USED_TIMERS` and use loop here, but it won't help that much in `enableInverter`